frontend/__tests__/containers/MetricsStep/MetricsStep.test.tsx

```diff
@@ -304,6 +304,26 @@ describe('MetricsStep', () => {
       ).toBeInTheDocument();
     });
 
+    it('should be render no pipeline container when get pipeline when selected future time ', async () => {
+      const MOCK_DATE_RANGE = {
```

Comment:
Should not mock coming dateTime, should mock current time to an old dateTime & select `mocked current dateTime` <`the old dateTime` < `real now()`